components/admin-panel/sections/ActivityLog/ActivityListItem.js

```diff
@@ -62,7 +62,7 @@ const ActivityListItem = ({ activity }) => {
               values={{
                 userName: !activity.individual ? (
                   <Span>
-                    <FormattedMessage id="user.unknown" defaultMessage="Unknown" />
+                    <FormattedMessage id="Unknown" defaultMessage="Unknown" />
```

Comment:
This won't work well with other languages; the translation for `Unknown` depends on the subject in some languages. E.g. in French if it's an unknown "author" than you'll translate with "Inconnu" but if it's an unknown "error" you'll say "Inconnue".

I suggest keeping the existing string as-is, and adding the word to `DUPLICATED_IGNORED_MESSAGES` in https://github.com/opencollective/opencollective-frontend/blob/44931662943074baa9796e8e8e064f6ef776158e/scripts/translate.js#L13